test/unit/tracks/text-track.test.js

```diff
@@ -25,7 +27,7 @@ TrackBaseline(TextTrack, {
   mode: 'disabled',
   label: 'English',
   language: 'en',
-  tech: defaultTech
+  tech: new TechFaker()
```

Comment:
its a suite of tests that runs on it's own, so it won't have access to `this.tech` as it is not in a `QUnit.test`